wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/romireference/RobotContainer.java

```diff
@@ -7,7 +7,9 @@
 import edu.wpi.first.wpilibj.GenericHID;
 import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.XboxController;
-import edu.wpi.first.wpilibj.examples.romireference.commands.TeleopArcadeDrive;
+import edu.wpi.first.wpilibj.examples.romireference.commands.ArcadeDrive;
+import edu.wpi.first.wpilibj.examples.romireference.commands.AutonomousDistance;
+//import edu.wpi.first.wpilibj.examples.romireference.commands.AutonomousTime;
```

Comment:
You need to either remove the commented out line here, or use a sendable chooser below so that you can pick between both auto routines

Reply:
(otherwise spotless will complain)

Reply:
Changed code to use chooser with SmartDashboard. All checks pass now.